tools/nuget2bazel/rules/RuntimeCoreGenerator.cs

```diff
@@ -44,7 +44,7 @@ private async Task Handle(StreamWriter f, Sdk sdk)
             await f.WriteLineAsync("def define_runtime():");
             await f.WriteLineAsync("    \"Declares runtime\"");
 
-            if (sdk.Packs != null)
+            if (sdk.Packs != null && !sdk.Version.StartsWith("6"))
```

Comment:
Why is this change needed?

Reply:
The aliases created here already appear as generated `core_stdlib_internal` target names in `stdlib/6.0.101.bzl` after running `nuget2bazel`.

There are no actual targets prefixed with `p1_...` to reference with the alias in the 6.0 generated code, since these DLLs are generated directly with the expected name: https://github.com/bazelbuild/rules_dotnet/pull/273/files#diff-82c7e5d7038182e73763f0843832c48dde3e3c0a374f0e1432d51d63964cf97fR1585.

This change removes the aliases since the `p1_` prefix does not exist. This only happens for code generated for the 6.0 runtime, and not for previous runtimes, where the `p1_` prefixes still exist.